etc/cherry-picker.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+sha=$1
+base=v1
+target=master
+dirname="/tmp/go-driver-$(openssl rand -hex 12)"
```

Comment:
Optional: Consider using `mktemp -d` instead, which honors the temp dir on different OSes.

Reply:
Done